frontend/src/components/editor/Editor.tsx

```diff
@@ -113,6 +286,14 @@ function Editor() {
 						minHeight: "100%",
 					}}
 				/>
+				{formatBarState.show && editorStore.cmView && (
+					<FormatBar
+						formatBarState={formatBarState}
+						setFormatBarState={setFormatBarState}
+						applyFormat={applyFormat}
+						cmView={editorStore.cmView}
```

Comment:
Should we pass `editorStore.cmView` to props?
It seems that we can access this varible in `FormatBar`.

(Maybe, it is the symptom you talked to me.)